### PR TITLE
Announcements

### PR DESCRIPTION
**Pull request by ckvan**
Removed in-development announcements.

### CODE REVIEW

File: app/controllers/announcements_controller.rb [hunk 1]
```diff
@@ -15,10 +15,8 @@ def new
 
   def create
     @announcement = Announcement.new(content_params)
-    if @announcement.save!
+    if @announcement.save! 
       redirect_to announcements_path
-    else
```
Comment (quichey) [inline app/controllers/announcements_controller.rb]: this part looked good. why get rid of it?

Comment (ckvan) [reply app/controllers/announcements_controller.rb]: Refactored later

**Review by quichey — COMMENTED** [review]
need to add authenticate_user! before each action:
"before_action :authenticate_user!"
like in TeamsController

File: app/controllers/announcements_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+class AnnouncementsController < ApplicationController
+  before_action :find_message, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @announcements = Announcement.all.order("created_at DESC")
+  end
+
+  def show
+    @announcement = Announcement.find(params[:id])
```
Comment (georgelzeng-zz) [inline app/controllers/announcements_controller.rb:9]: show is same function as find_message, and which is listed as a before_action for show. repetitive

**Pull request merged**